week-4/Homework/mandatory/2-writers.js

```diff
@@ -5,14 +5,10 @@ Did you know you can also have an Array of Objects? You might think "This is mad
 it is quite a frequent combination. Just think about what benefits we can get from this construct.
 
 An object lets you store multiple values in a single variable, then you can store complex objects in an array.
-Let's assume you have a list of data about people names and their birthday and you would like to print each name
-with corresponding birthday together. Storing these pieces of information in different arrays and then pairing them up
-makes the iteration unnecessarily complicated, code will be less intuitive, needs extra cognitive effort to
-reason about and last but not least it can be error-prone (for example, you pick up the wrong birthday to a name).
 
-In this exercise you will practice how to access to Objects stored in an Array and their properties. You already know 
-different ways of looping through Arrays, it won't be different in this case. The only extra step is that you have to 
-use values inside Objects.
+Let's assume you have a list of data about people names and their birthday and you would like to print each name with corresponding birthday together. Storing these pieces of information in different arrays and then pairing them up makes the iteration unnecessarily complicated, code will be less intuitive, needs extra cognitive effort to reason about and last but not least it can be error-prone (for example, you pick up the wrong birthday to a name).

```

Comment:
Could you re-write this to use simpler English? I'm worried some of our student might not understand it 

e.g. (iteration, cognitive, error-prone, intuitive, corresponding)

Reply:
Breaking this up on to multiple bullet points would help as well though